pkg/agent/config/node_config.go

```diff
@@ -23,6 +23,7 @@ import (
 
 const (
 	DefaultTunPortName = "tun0"
+	AutoAssignedOFPort = 0
```

Comment:
This one should be defined in ovsconfig, but not here, as it is a predefined value in OVS.

Reply:
Sure, it's better to move the const to ovsconfig. Done.

And actually it's not a predefined value in OVS. We just use a invalid request port number as param and OVS will assign port number automatically.